tasks/eth/016-granite-protocol-versions/VALIDATION.md

```diff
@@ -0,0 +1,60 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of setting the recommended protocol version.
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state changes occur.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## State Overrides
+
+The following state overrides should be seen:
+
+### `0x8062AbC286f5e7D9428a0Ccb9AbD71e50d93b935` (`ProtocolVersions`)
+
+We override the ownership of the PV contract (task#011).
```

Comment:
This is an outdated reference.

Reply:
leftover copy n paste from previous pv update. Fixed.